### PR TITLE
Define -second-item through to -fifth-item

### PR DESCRIPTION
**Pull request by Wilfred**
cl-second is a very useful function that has no dash
equivalent. Define functions for accessing the second, third, fourth
and fifth items of a list.

### CODE REVIEW

File: dash.el [hunk 1]
```diff
@@ -576,13 +576,43 @@ Alias: `-any'"
...
+\(fn LIST)")
+
+(defalias '-third-item 'caddr
+  "Return the third item of LIST, or nil if LIST is too short.
+
+See also: `-fourth-item'.
+
+\(fn LIST)")
+
+(defalias '-fourth-item 'cadddr
```
Comment (Fuco1) [inline dash.el]: Cadddr comes from `cl` which we don't use.  Is it built-in?  I'm not sure.

I'm not sure if copying the definition would be better though, e.g.

```elisp
(defun cl-cadddr (x)
  "Return the `car' of the `cdr' of the `cdr' of the `cdr' of X."
  (declare (compiler-macro cl--compiler-macro-cXXr))
  (car (cdr (cdr (cdr x)))))
``` 

could be just copied into dash.

**Comment by Wilfred** [comment]
Rebased and fixed as suggested. ✨

**Comment by Fuco1** [comment]
It would also be cool if we made them into generalized variables so we could use `setf` with them.  But that's probably for a separate PR.  I'm going to merge this one, thanks!

**Comment by Fuco1** [comment]
Oh this info thing is hell... I think we'll need to find a better workflow than people changing the file in PRs... ideally some bot or CI could do that and push an update.

**Comment by Wilfred** [comment]
OK, rebased :)

I think dash.el would be more convenient if it didn't set up the commit hooks. Creating a PR without updating dash.info is fine (you could always generate it after), but getting a merge conflict is a little more hassle for new contributors.

**Comment by Fuco1** [comment]
Does it setup the commit hooks though?  I don't have any.  I think you have to do it manually somehow.

**Comment by Wilfred** [comment]
You're right. Looks like I followed the instructions in the readme months ago:

> I highly recommend that you install these as a pre-commit hook, so that
the tests are always running and the docs are always in sync:
>
>      cp pre-commit.sh .git/hooks/pre-commit

**Pull request merged**